web/business-registry-dashboard/app/components/AsyncComboBox/index.vue

```diff
@@ -283,9 +283,9 @@ function handleInput (e: Event) {
     <UInput
       id="business-search-input"
       ref="inputRef"
+      v-model="query"
```

Comment:
I believe the input wasn't properly reactive in this version of NuxtUI.

Ref: https://ui2.nuxt.com/components/input